textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/neextrator/NamedEntityExtractor.java

```diff
@@ -0,0 +1,64 @@
+package edu.uci.ics.textdb.dataflow.neextrator;
+
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.dataflow.IOperator;
+import edu.uci.ics.textdb.common.exception.DataFlowException;
+
+
+/**
+ * Created by Sam Hong on 16/4/21.
+ */
+public class NamedEntityExtractor implements IOperator{
+
+
+    private IOperator sourceOperator;
+    private ITuple sourceTuple;
+
+
+    public NamedEntityExtractor(IOperator operator) {
+        this.sourceOperator = operator;
+    }
+
+
+    /**
+     * @about Opens Named Entity Extractor
+     */
+    @Override
+    public void open() throws Exception {
+        try {
+            sourceOperator.open();
+        } catch (Exception e) {
+            e.printStackTrace();
+            throw new DataFlowException(e.getMessage(), e);
+        }
+    }
+
+    /**
+     * @about Return all named entities that recognized in the a document.
```

Comment:
Please write high-quality comments.  For instance:

"recognized" -> "are recognized".

"the a" -> "a"